Tools/TestWebKitAPI/Tests/WebCore/ContentExtensions.cpp

```diff
@@ -3061,10 +3062,10 @@ TEST_F(ContentExtensionTest, Serialization)
         { { ModifyHeadersAction::ModifyHeaderInfo::SetOperation { "key2"_s, "value2"_s } } }
     }, {
         { { ModifyHeadersAction::ModifyHeaderInfo::RemoveOperation { String::fromUTF8("💩") } } }
-    } };
+    }, 2 };
     Vector<uint8_t> modifyHeadersBuffer;
     modifyHeaders.serialize(modifyHeadersBuffer);
-    EXPECT_EQ(modifyHeadersBuffer.size(), 55u);
+    EXPECT_EQ(modifyHeadersBuffer.size(), 59u);
```

Comment:
We should probably increment CurrentContentRuleListFileVersion when changing the serialization so we don't try to deserialize existing files using the new format.